aries-site/src/examples/foundation/icons/IconCoreExample.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card, CardHeader, CardBody, Grid, Heading, Paragraph } from 'grommet';
+import {
+  Add,
+  Close,
+  Download,
+  Edit,
+  Filter,
+  FormDown,
+  FormUp,
+  Notification,
+  Search,
+  Subtract,
+  Trash,
+} from 'grommet-icons';
+
+const CoreIcon = ({ Icon, name, guidance }) => (
+  <Card elevation={false}>
+    <CardHeader direction="column" align="stretch" gap="small">
```

Comment:
```suggestion
    <CardHeader direction="column" align="stretch" gap="small" pad={{ top: 'medium', horizontal: 'medium', bottom: 'small' }} >
```

Card designs call for 12px space between header and body description.

Reply:
Does that mean that all uses of Card have to customize padding in this way? I would have expected this to be built into the theme.

Reply:
Currently it's not built into the theme to know how to handle when CardHeader, CardBody, CardFooter are present. Would be nice if we could make some enhancements so the caller doesn't have to worry about it.

Reply:
We have padding set in the [HPE theme](https://github.com/grommet/grommet-theme-hpe/blob/31e52ccc1aa211be75e63eed8d73d1f03580a42a/src/js/themes/hpe.js#L387) for body, footer, and header. Am I misunderstanding? Or are you saying the values in the theme don't align with the designs?

Reply:
^^ @halocline As Card patterns have become a little more fleshed out, the pad values that we having in the theme apply on all sides of each of the card elements (body, footer, header) which results often in too much padding on the sides where body + header meet or body + footer meet, etc.

Proposing as Card patterns become more fleshed out that we can be more intelligent (under the hood) of how the padding gets applied to card elements when body + footer, body + header etc. are in play.

Reply:
Is there any consistency to the desired spacing? Can we place the responsibility on `card.container.gap` and remove vertical pad from body, footer, header?

Reply:
I think that's a possibility and we could just apply horizontal spacing to the individual elements, but a bit more investigation would need to be done to confirm (and our design variants should be ironed out first). I think this would be important follow-on work to the Card exploration I had done. I filed a ticket for it: https://github.com/grommet/hpe-design-system/issues/2521